deploy/charts/rook-ceph/values.yaml

```diff
@@ -77,9 +77,14 @@ containerSecurityContext:
 # -- If true, loop devices are allowed to be used for osds in test clusters
 allowLoopDevices: false
 
+# ceph-csi operator overrides
 ceph-csi-operator:
-  nameOverride: ceph-csi-operator
-  fullnameOverride: ceph-csi-operator
+  nameOverride: ceph-csi
+  fullnameOverride: ceph-csi
+controllerManager:
+  manager:
+    env:
+      csiServiceAccountPrefix: ceph-csi-
```

Comment:
@Madhu-1 How can we apply the service account prefix as `ceph-csi-` instead of `ceph-csi-operator-`? During a helm install, I could not find a way to override this setting. In my helm testing, the env var was always:
```yaml
- name: CSI_SERVICE_ACCOUNT_PREFIX
  value: ceph-csi-operator-
```
Is this a bug in the csi operator helm chart, or what setting should I override from rook's values.yaml?

After helm install, if I manually edited that env var csi operator's deployment, the issue was fixed. So the question is just how to apply this setting in the helm chart.

Reply:
Yes, I have added the configuration in values.yaml in my other PR
```yaml
ceph-csi-operator:
  nameOverride: ceph-csi
  fullnameOverride: ceph-csi
  controllerManager:
    manager:
      env:
        csiServiceAccountPrefix: "ceph-csi-"
```